### ISSUE TITLE
Customizable User Added Options

### ISSUE THREAD

**Issue by TestCort** [fbrettnich/easypoll-bot#17]
This is a memo regarding the Discord Bot EasyPoll and it's possibilities.
Upon suggestion, I started to write this here.
The idea is, as the title says, that users could add their own options to the presented polls.
The options would be added post creation of the poll itself and will be able to be acted upon
(such as voting) by the other users when they see them. They would not differ by function from 
the original poll options provided by the polls creator.

Example: 
Let's say I create a poll "What's your favorite time of day?"
And among the poll options are "Morning, Noon, Evening"
One of the users sees the poll and his favorite time is "Midnight" for instance.
The idea is that the user would be able to add the option "Midnight" 
 after the poll's creation and that other users after that action can see and vote on it.
 
 In conclusion, 
 I am not sure if this idea is even viable. One example for such a function would 
 be Facebook's polls on various groups and pages. As much as I am not well versed
 in Python, I hope you will be able to make something of this request as this seems to 
 be a great tool for any Discord server.
